composer.json

```diff
@@ -15,19 +15,29 @@
         "drupal/address": "~1.2",
         "drupal/admin_toolbar": "~1.20",
         "drupal/allowed_formats": "~1.1",
+        "drupal/better_normalizers": "1.x-dev",
+        "drupal/block_blacklist": "~1.0",
         "drupal/chosen": "~2.4",
         "drupal/components": "~1.0",
         "drupal/config_readonly": "~1.0@beta",
         "drupal/core": "~8.6",
         "drupal/default_content": "^1.0@alpha",
         "drupal/diff": "^1.0@RC",
         "drupal/ds": "~3.3",
+        "drupal/environment_indicator": "^3.0",
+        "drupal/file_mdm": "~1.1",
         "drupal/google_analytics": "~2.1",
         "drupal/honeypot": "^1.27",
-        "drupal/imagemagick": "^3.0",
-        "drupal/menu_block": "^1.5",
+        "drupal/imagemagick": "~3.0@alpha",
+        "drupal/layout_builder_modal": "~1.0@alpha",
+        "drupal/layout_builder_restrictions": "~2.2",
+        "drupal/layout_library": "~1.0@beta",
+        "drupal/menu_admin_per_menu": "^1.0",
+        "drupal/menu_block": "dev-1.x",
```

Comment:
What was wrong with version 1.5? Can we lock this to a hash?

Reply:
it doesnt have the "follow active parent" https://www.drupal.org/project/menu_block/issues/2756675
We might use it, we might not. :man_shrugging: